<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Arial&quot;;
background-color: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>                     Procurar Dados </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite o NB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>69</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Tela principal/Gerar pdf.png);
background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Tela principal/sair.jpg);
background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Tela principal/Procurar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Tela principal/INSS.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo de Acontecimento</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fonte de Ação</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Espécie</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Número de Protocolo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome Segurado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CPF Segurado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NIT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CEP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rua</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bairro</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Estado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Municipio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome Responsavel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Dívida</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Calculado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor de Entrada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parcelas Pagas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total de Parcelas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data da Cobrança</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data do Calculo</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Tela principal/Apagar.png);
background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver PDFs Gerados</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite sua Matricula</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>tableWidget</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>lineEdit_2</zorder>
 </widget>
 <resources>
  <include location="../IMAGENS/img.qrc"/>
 </resources>
 <connections/>
</ui>
